src/06-loops.js

```diff
@@ -117,7 +163,9 @@ function targetCount(array, target) {}
  *  firstIndexFound([ "left", "right", "left" ], "up");
  *  //> -1
  */
-function firstIndexFound(array, target) {}
+function firstIndexFound(array, target) {
+  return array.findIndex(item => item==target)
+}
```

Comment:
this can also be solved with
`function firstIndexFound(array, target) {

  for (var i in array) {
    if (array[i]==target) return parseInt(i);
  }
  return -1
}`